packages/react-sdk/src/components/box.ws.ts

```diff
@@ -14,7 +14,8 @@ const meta: WsComponentMeta = {
   label: "Box",
   Icon: SquareIcon,
   presetStyle,
-  props: props as MetaProps,
+  props: MetaProps.parse(props),
+  initialProps: ["tag"],
```

Comment:
We used to treat properties that have a default value like initial props, now only:
 - explicitly in `initialProps`
 - `required: true`